app/models/regex_pattern.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module RegexPattern
```

Comment:
This might be better inside an additional module? `RegexPattern` in isolation doesnt tell me much of what this actually is

Reply:
This class and the next one would both benefit with some heredocs for the classes themselves to help set context on purpose and use